### PR TITLE
Removes miniscope from grenade launchers

### PR DESCRIPTION
**Pull request by RikuTheKiller**
## About The Pull Request

The definition of a xeno cope PR, though I've already held the opinion that B17 or gl should be nerfed months ago.

This PR removes your ability to attach miniscopes to grenade launchers.
## Why It's Good For The Game

Anyways, miniscope with gl allows you to essentially turn the grenade launcher into, well, a sniper. Why is this a problem? Well, the longer flight time of the grenade allows it to essentially turn into a wider AoE RR since it's detonation timer is much lower than normal. It also allows the gl to have straight up uncanny chasing power, which combined with it's knockback makes it plain OP and a pain to deal with. (Resulting in massive xeno cope.)

Now, while you may think you can counter a B17 that's using gl by just going in cqc, well, you can't! They'll pull out a shotgun and use their insane armor values (75 by the way.) to take 5 hits for free before their health HUD even shows up! All while destroying your ass with a shotgun. It's no surprise this setup always fucks the entire hive over whenever it shows up. However, I can't simply nerf B17 due to how it's current balancing is reliant on the fact it's above tyr2 in armor values.

TLDR; RR with a scope.
## Changelog
:cl:
balance: Miniscopes can no longer be attached to grenade launchers.
/:cl:

### CODE REVIEW

**Comment by Pariah919** [comment]
RR has a scope though?

**Comment by RikuTheKiller** [comment]
> RR has a scope though?

Correction, scoped insta-shot multi-shot RRs.

**Comment by TiviPlus** [comment]
idk why you write a rant about knockback and b17 on this PR but I dont see enough reason to remove this
